metals/src/main/scala/scala/meta/internal/metals/FindTextInDependencyJarsRequest.scala

```diff
@@ -0,0 +1,6 @@
+package scala.meta.internal.metals
+
+case class FindTextInDependencyJarsRequest(
```

Comment:
Can we add some scaladocs to this. From the outside it might not be clear what exactly `mask` is supposed to be here or even content really.